kivy/core/window/window_sdl2.py

```diff
@@ -714,7 +714,8 @@ def request_keyboard(self, callback, target, input_type='text'):
 
     def release_keyboard(self, *largs):
         super(WindowSDL, self).release_keyboard(*largs)
-        self._win.hide_keyboard()
+        if self._system_keyboard.widget:
+            self._win.hide_keyboard()
         self._sdl_keyboard = None
```

Comment:
I am actually not sure why i put the system keyboard widget there, i must have been looking at the wrong code when i was rushing to complete my widget. It works because it always returns None